internal/lib/checkpoint.go

```diff
@@ -37,11 +37,44 @@ func (c *ContainerServer) ContainerCheckpoint(
 		return "", fmt.Errorf("not able to read config for container %q: %w", ctr.ID(), err)
 	}
 
+	if err := c.runtime.UpdateContainerStatus(ctx, ctr); err != nil {
+		return "", err
+	}
+
```

Comment:
is this protection against the container stopping before being paused?

Reply:
It was not necessary. I removed it.